server/fam_api.tf

```diff
@@ -35,18 +35,18 @@ data "aws_iam_policy_document" "fam_api_lambda_exec_policydoc" {
   }
 }
 
-resource "aws_iam_role" "fam_api_lambda_exec" {
-  name = "fam_api_serverless_lambda_role"
-  assume_role_policy = data.aws_iam_policy_document.fam_api_lambda_exec_policydoc.json
+resource "aws_iam_role" "fam-api_lambda_exec" {
+  name = "fam-api_serverless_lambda_role"
+  assume_role_policy = data.aws_iam_policy_document.fam-api_lambda_exec_policydoc.json
 }
 
-resource "aws_lambda_function" "fam_api" {
-  filename      = "${path.module}/fam_api/function.zip"
-  function_name = "fam_api"
-  role          = aws_iam_role.fam_api_lambda_exec.arn
+resource "aws_lambda_function" "fam-api" {
+  filename      = "fam-api.zip"
```

Comment:
Just for your reference.
I am very very new to Terraform, but I was checking something yesterday that Terraform itself seems to have a ability to create zip file itself. Maybe later this can be improved:
Reference: https://registry.terraform.io/providers/hashicorp/archive/latest/docs/data-sources/archive_file

Reply:
@ianliuwk1019 True, I saw this as well. 
Lets take note of this for future enhancement, I will add it to my work.

Thanks.